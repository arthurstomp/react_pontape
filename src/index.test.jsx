import React from 'react';
import Index from './index'
import renderer from 'react-test-renderer';

test('Render "Hello"', () => {
  const component = renderer.create(
    <Index />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
