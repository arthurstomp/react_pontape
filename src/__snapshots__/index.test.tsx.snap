// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render "Hello" 1`] = `
Array [
  <h1>
    Index 
    hello
  </h1>,
  <button
    onClick={[Function]}
  >
    Increment Count
  </button>,
]
`;
